external/versions.cmake

```diff
@@ -21,8 +21,8 @@ set(TA_TRACKED_MADNESS_PREVIOUS_TAG 2353a30fb3257e2e7f5133975d909d8346dc7192)
 set(TA_TRACKED_MADNESS_VERSION 0.10.1)
 set(TA_TRACKED_MADNESS_PREVIOUS_VERSION 0.10.1)
 
-set(TA_TRACKED_BTAS_TAG 8418021a283000ada8edd4994bd81f35652d4b49)
-set(TA_TRACKED_BTAS_PREVIOUS_TAG f5a50ebf11b5cdc219ff4c06e3652eea2cd6c697)
+set(TA_TRACKED_BTAS_TAG a6476624db407c1558e8f1fa0750498a31743e06 )
```

Comment:
since you did not have git-hooks installed you need to update INSTALL.md with the new BTAS tag

Reply:
Sorry about these issues around git-hooks.  I have the MPQC hooks installed but realize I needed the TA hooks as well. I them installed now, will git-hooks automatically maintain the BTAS tag in INSTALL.md now?